### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by qyx268**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-# magcalc
+Computing galaxy spectral energy distributions (SEDs) following stellar population synthesis generated by [STARBURST99](https://www.stsci.edu/science/starburst99/docs/default.htm). This includes the spectrum, the rest-frame photometry and observed magnitudes within the HST or JWST NIRCam filters. See `examples/` for demonstration.
```
Comment (EMventura) [inline README.md:2]: Here I would add:

Pop. III spectra are taken from Raiter et al. (2010) assuming 4 different Pop. III IMFs: 
Salpeter [1 - 500] Msun, Salpeter [50 - 500] Msun, Log-Normal with characteristic mass = 10 Msun and Log-Normal with characteristic mass = 60 Msun (see Ventura et al. 2024 for more details).

**Review by EMventura — COMMENTED** [review]
Did you add the Pop. III spectra? I can't find those!

File: README.md (hunk 1 above)
Comment (qyx268) [reply README.md:2]: I was waiting for Simon's comment on where to put them

Comment (qyx268) [reply README.md:2]: I just added your note

Comment (qyx268) [reply README.md:2]: also just added the spectra

**Comment by smutch** [comment]
I'll try to take a look at this on Friday arvo. Sorry for the delay

**Comment by qyx268** [comment]
> I'll try to take a look at this on Friday arvo. Sorry for the delay

No problem. Then I had a PR to have this into your sector. I closed the PR because I wanted to wait and see if I would have the permission to deploy my sector into the collaboration. So I thought we could reopen the PR if we ended up still needing you to do do the deployment. 

Anyway, you might reopen the PR and take a look before merging into yours.

**Pull request merged**